/**:
  ros__parameters:
    p0: [90.51855671643054, -21.17744166007899]
    gnss0: [36.08313259166667, 140.07603255833334]
    p1: [-378.2120422862277, 262.14895598113344]
    gnss1: [36.080194641666665, 140.08080029833334]
    a: 6378137.0
    F: 298.257222
    m0: 0.999
    theta_offset: -276.0
    ignore_th_cov: 16.0
